### PR TITLE
Units - Add Combat Uniform (Plaid Dark Blue, Black, Black)

### PR DESCRIPTION
**Pull request by JoramD0**
**When merged this pull request will:**
- Add `Combat Uniform (Plaid Dark Blue, Black, Black)` (rolled and long sleeve versions).
- ~~Sort some stuff alphabetically.~~
- Fix some comments missing on rolled sleeve variants.

Textures/images made by @GilleeDoo .

### CODE REVIEW

**Comment by JoramD0** [comment]
Tested in-game, no issues

**Comment by jonpas** [comment]
> Sort some stuff alphabetically.

Is the order the same in config files as in `CfgPatches` now or different?

**Comment by JoramD0** [comment]
> > Sort some stuff alphabetically.
> 
> Is the order the same in config files as in `CfgPatches` now or different?

I don't know, just sorted that bit I did, can go over it tomorrow.

**Comment by jonpas** [comment]
I don't think it was ever meant to be sorted, just same order in every place. But not sure how true that ended up being.

File: addons/units/uniforms/combat_long_plaid.hpp [hunk 1]
```diff
@@ -60,6 +60,16 @@ class CLASS(Uniform_Combat_LS_CDBS_GP_TB): CLASS(Uniform_Base) { // CDB = Plaid
         uniformClass = QCLASS(Unit_Combat_LS_CDBS_GP_TB);
     };
 };
+class CLASS(Uniform_Combat_LS_CDBS_BP_BB): CLASS(Uniform_Base) { // CDB = Plaid Dark Blue
+    scope = 2;
+    author = "GilleeDoo";
+    displayName = CSTRING(Uniform_Combat_LS_CDBS_BP_BB);
+    picture = QPATHTOF(UI\uniform_combat_ls_cdbs_bp_ca.paa);
```
Comment (Mike-MF) [inline addons/units/uniforms/combat_long_plaid.hpp:67]:
```suggestion
    picture = QPATHTOF(UI\uniform_combat_ls_cdbs_bp_ca.paa);
```

This file doesn't exist.

File: addons/units/uniforms/combat_rolled_plaid.hpp [hunk 2]
```diff
@@ -60,7 +60,17 @@ class CLASS(Uniform_Combat_RS_CDBS_GP_TB): CLASS(Uniform_Base) {
         uniformClass = QCLASS(Unit_Combat_RS_CDBS_GP_TB);
     };
 };
-class CLASS(Uniform_Combat_RS_CLBS_GP_BB): CLASS(Uniform_Base) {
+class CLASS(Uniform_Combat_RS_CDBS_BP_BB): CLASS(Uniform_Base) { // CDB = Plaid Dark Blue
+    scope = 2;
+    author = "GilleeDoo";
+    displayName = CSTRING(Uniform_Combat_RS_CDBS_BP_BB);
+    picture = QPATHTOF(UI\uniform_combat_rs_cdbs_bp_ca.paa);
```
Comment (Mike-MF) [inline addons/units/uniforms/combat_rolled_plaid.hpp:67]:
```suggestion
    picture = QPATHTOF(UI\uniform_combat_rs_cdbs_bp_ca.paa);
```

Neither does this one, although normally we only do the ui image for the LS variant and use it on both as people don't pay attention to them anyway.

**Review by Mike-MF — CHANGES_REQUESTED** [review]

File: addons/units/uniforms/combat_long_plaid.hpp (hunk 1 above)
Comment (Mike-MF) [reply addons/units/uniforms/combat_long_plaid.hpp:67]: Asked Mick to make it, will push it when he sends it to me.

Comment (JoramD0) [reply addons/units/uniforms/combat_long_plaid.hpp:67]: I should have them, thought I pushed them already, will do tomorrow

Comment (JoramD0) [reply addons/units/uniforms/combat_long_plaid.hpp:67]: I forgot to push them...

**Review by Mike-MF — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**